client/src/main/scala/com/gu/acquisition/services/GAService.scala

```diff
@@ -91,6 +91,7 @@ private[services] class GAService(implicit client: OkHttpClient)
           "cd25" -> acquisition.labels.exists(_.contains("REUSED_EXISTING_PAYMENT_METHOD")), // usedExistingPaymentMethod
           "cd26" -> acquisition.labels.exists(_.contains("gift-subscription")), // gift subscription
           "cd27" -> productCheckout,
+          "cd30" -> acquisition.labels.exists(_.contains("corporate-subscription")), // corporate subscription
```

Comment:
I know you didn't start the trend, but do we really need the comment?
I assume the comment refers to "cd30" but I'd rather put it into a named constant than a comment